<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFramelessWidget</class>
 <widget class="QWidget" name="frmFramelessWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>弹出</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDialog">
      <property name="text">
       <string>Dialog</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnWidget">
      <property name="text">
       <string>Widget</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMainWindow">
      <property name="text">
       <string>MainWindow</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
